### DISCUSSION TITLE
Resolved RTSP issue with Pan v2

### DISCUSSION THREAD

**Discussion by jaggles** [gtxaspec/wz_mini_hacks#351] [General]
Just a quick post in case anyone else encounters the same issue I did. I couldn't get RTSP working on my Pan v2, and I'd tried all of the fixes listed here. The only thing that worked for me was upgrading the stock Wyze firmware.

I was originally on 4.49.0.*

I upgraded, it took me to 4.49.1.20 but that still didn't fix the issue. I noticed that there was immediately another firmware update available to take me to 4.49.1.47, and now RTSP is working.

Notes: I didn't need to do anything with wz_mini_hacks.. I just left the SD card in and did the firmware updates from the Wyze app.

I do see that the Wyze app currently isn't able to connect to my camera. I'm not bothering to troubleshoot this, though, as I don't need the Wyze app since I'm only using RTSP anyway.

**Comment by jaggles** [comment]
UPDATE: The Wyze app now works. Seems to have just resolved itself after a few minutes.

**Comment by rkrueger11** [comment]
Any chance you have a link or can send me the older firmware file for the pan v2?

**Reply by akivacp** [reply]
4.49.1.47 for Pan v2
https://download.wyzecam.com/firmware/panV2/recovery_wcpv2_4.49.1.47.zip
